src/Util/AdminAclUserManagerInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Util;
+
+/**
+ * @author Mathieu Petrini <mathieupetrini@gmail.com>
+ */
+interface AdminAclUserManagerInterface
+{
+    /**
+     * Batch configure the ACLs for all objects handled by an Admin.
+     */
+    public function findUsers(): \Traversable;
```

Comment:
isnt it better to use iterable?

Reply:
I would have use `array` as return typehint ; like we did anywhere else in our code.

Reply:
When related to the persistence, we are starting to use Collection too.

Reply:
Even when persistence are returning `Collection` we're transforming them to `array`
https://github.com/sonata-project/SonataAdminBundle/blob/master/src/Datagrid/SimplePager.php#L68

If we start using `iterable` which is not a bad idea, it's maybe better to change it too for others signatures ?

Reply:
In any case, `iterable` or `array` seems better than `\Traversable`